src/fetch/payouts.py

```diff
@@ -103,21 +111,28 @@ def from_series(cls, frame: Series) -> RewardAndPenaltyDatum:
         )
         solver = frame["solver"]
         reward_target = frame["reward_target"]
-        bonding_pool = frame["pool_address"]
         if reward_target is None:
-            logging.warning(f"solver {solver} without reward_target. Using solver")
+            logging.warning(f"Solver {solver} without reward_target. Using solver")
             reward_target = solver
 
+        buffer_accounting_target = frame["buffer_accounting_target"]
+        if reward_target is None:
+            logging.warning(
+                f"Solver {solver} without buffer_accounting_target. Using solver"
+            )
+            buffer_accounting_target = solver
+
         return cls(
             solver=Address(solver),
             solver_name=frame["solver_name"],
             reward_target=Address(reward_target),
-            bonding_pool=Address(bonding_pool),
+            buffer_accounting_target=Address(buffer_accounting_target),
             slippage_eth=slippage,
             primary_reward_eth=int(frame["primary_reward_eth"]),
             primary_reward_cow=int(frame["primary_reward_cow"]),
             quote_reward_cow=int(frame["quote_reward_cow"]),
-            service_fee=bool(frame["service_fee"]),
+            service_fee=Fraction(frame["service_fee"]),
```

Comment:
I did a dry run for the past week with the ignore slippage flag being on. I saw this:

`ValueError: cannot convert NaN to integer ratio`

Reply:
I have to think a bit about how to restructure this. The rest of the code should always return a valid value, as the default option is not available in this function.

Reply:
One issue seems to be that the service fee query is a bit unstable. It does not return a value for the old rizzolver prod account, for example. This then gives unexpected results when rerunning the query. I will set a default value of 0% for now.

Reply:
Oh good catch. Thanks for investigating. yeah, default of 0% is the only reasonable thing to do i think

Reply:
I added another log for this.